homeassistant/components/keymitt_ble/switch.py

```diff
@@ -0,0 +1,36 @@
+"""Switch platform for MicroBot."""
+from __future__ import annotations
+
+from typing import Any
+
+from homeassistant.components.switch import SwitchEntity
+
+from .const import DEFAULT_NAME, DOMAIN
+from .entity import MicroBotEntity
+
+
+async def async_setup_entry(hass, entry, async_add_entities):
+    """Set up MicroBot based on a config entry."""
+    coordinator: MicroBotDataUpdateCoordinator | None = hass.data[DOMAIN][entry.entry_id]
```

Comment:
```suggestion
    coordinator: MicroBotDataUpdateCoordinator = hass.data[DOMAIN][entry.entry_id]
```

This should never be None here